pkgs/top-level/all-packages.nix

```diff
@@ -21263,6 +21263,14 @@ with pkgs;
 
   myserver = callPackage ../servers/http/myserver { };
 
+  napalm = with python3Packages; toPythonApplication (
+    napalm.overridePythonAttrs (attrs: {
+      propagatedBuildInputs = attrs.propagatedBuildInputs ++ [
+        napalm-hp-procurve
```

Comment:
A comment explaining why this was done might be nice

Reply:
Explanation, or rather an invitation to add more community plugins?

Reply:
done